Code/Editor/EditorFramework/CodeGen/CppProject.cpp

```diff
@@ -1001,6 +1001,16 @@ namespace
     inout_json[sName] = sValue;
     inout_modified = ezCppProject::ModifyResult::MODIFIED;
   }
+
+  void Remove(ezVariantDictionary& inout_json, ezStringView sName, ezCppProject::ModifyResult& inout_modified)
+  {
+    if (inout_json.Contains(sName))
+    {
+      inout_json.Remove(sName);
```

Comment:
Doesn't this remove function already return whether something was removed?

Reply:
Oh it does! Much better, thanks!